index.html

```diff
@@ -29,5 +29,142 @@ <h1>Charlie Giorgania</h1>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-3.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-4.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div><img class="works-img" src="./Media/work-img-5.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div>
+    <img class="works-img" src="./Media/work-img-6.svg" alt="Man typing on a laptop">
+    <div class="works-box">
+      <h3>Project name goes here</h3>
+      <div class="lang-container">
+        <div class="lang-box">HTML/CSS</div>
+        <div class="lang-box">Ruby on Rails</div>
+        <div class="lang-box">JavaScript</div>
+      </div>
+      <a href="#" rel="noopener noreferrer" class="works-link">
+        <div class="works-button">
+          <span>See this project</span>
+          <div class="works-arrow"></div>
+      </div>
+      </a>
+    </div>
+</section>
+<section class="about">
+  <h2>About</h2>
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+    veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
+    eiusmod
+  </p>
+  <div class="about-container">
+    <div class="cv-box">
+      <span>Get my CV</span>
+      <div></div>
+    </div>
```

Comment:
- [ ] This should be a button and The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed.

Reply:
Satisfied.